src/toolchain.rs

```diff
@@ -725,7 +725,11 @@ impl RustupToolchain {
         );
 
         // Small sanity check.
-        let rustc_path = dir.join("bin/rustc");
+        #[cfg(not(target_os = "windows"))]
+        let rustc_path = "rustc";
+        #[cfg(target_os = "windows")]
+        let rustc_path = "rustc.exe";
+        
         if !rustc_path.is_file() {
```

Comment:
Encountered the same issue. #13 looks good to me, but I just wanted to point out that your fix is missing 'bin' in the path.

Reply:
Yeah, closing this.